jsk_topic_tools/CMakeLists.txt

```diff
@@ -226,5 +233,4 @@ install(DIRECTORY test
 )
 install(DIRECTORY cmake config launch sample
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-  USE_SOURCE_PERMISSIONS
```

Comment:
@sktometometo could you revert this line?
because now `sample/` directory contains
https://github.com/jsk-ros-pkg/jsk_common/blob/master/jsk_topic_tools/sample/simple_image_transport.py
so we need to use `USE_SOURCE_PERMISSIONS` to pass CI 
https://github.com/jsk-ros-pkg/jsk_common/runs/6752693789?check_suite_focus=true

Reply:
I don't know why I remove this line. anyway I added this line again.